### PR TITLE
Update issue in GitHub Games repo

### PR DESCRIPTION
**Pull request by beardofedu**
Issue `#2` that is created in the GitHub Games repository incorrectly tells the student that the URL in the README **and** the description are incorrect. The URL in the description is correct, this PR removes that text from the issue body and the issue title. 

image

### CODE REVIEW

**Comment by beardofedu** [comment]
@parkerbxyz , we would just need to modify this part of the repo creation script to address @amyschoen 's ask, correct? 

`\"homepage\": \"${PAGES_URL}\",`

**Comment by parkerbxyz** [comment]
> \"homepage\": \"${PAGES_URL}\",

Yes, but I think we’ll need to make additional modifications for GitHub Enterprise Server compatibility. I’m looking into it now.

**Comment by parkerbxyz** [comment]
I’ve updated the script to replace the student’s Pages URL in the repo description with the template repo’s Pages URL. I’ve also changed the issue text back to what it was to reflect this change. I haven’t done any testing yet, but made several changes to the script to make sure these changes would be compatible with both github.com and GHES. 

@amyschoen could you please test these changes on GHES?

**Comment by amyschoen** [comment]
@parkerbxyz, the changes work on GHES, but there is a flaw.

```
if [[ "$ROOT_URL" == "github.com" ]]; then
    echo "Creating on GitHub"
    STUDENT_PAGES_URL="https://${CLASS_ORG}.github.io/${STUDENT_REPO}/"
    TEMPLATE_PAGES_URL="https://${CLASS_ORG}.github.io/${STUDENT_REPO}/"
  else
    echo "Creating on GitHub Enterprise Server"
    STUDENT_PAGES_URL="https://${ROOT_URL}/pages/${CLASS_ORG}/${STUDENT_REPO}/"
    TEMPLATE_PAGES_URL="https://${ROOT_URL}/pages/${CLASS_ORG}/${STUDENT_REPO}/"
  fi
```
You'll notice that both the student URL and the template URL are identical.

File: script/create-game-repos [hunk 1]
```diff
@@ -109,17 +114,19 @@ function CreateRepo() {
   # Check the root #
   ##################
   if [[ "$ROOT_URL" == "github.com" ]]; then
-    echo "Creating on GITHUB"
-    PAGES_URL="https://${CLASS_ORG}.github.io/${STUDENT_REPO}/"
+    echo "Creating on GitHub"
+    STUDENT_PAGES_URL="https://${CLASS_ORG}.github.io/${STUDENT_REPO}/"
+    TEMPLATE_PAGES_URL="https://${CLASS_ORG}.github.io/${STUDENT_REPO}/"
```
Comment (amyschoen) [inline script/create-game-repos:119]:
```suggestion
    TEMPLATE_PAGES_URL="https://${CLASS_ORG}.github.io/${$TEMPLATE_REPO}/"
```

File: script/create-game-repos [hunk 2]
```diff
@@ -109,17 +114,19 @@ function CreateRepo() {
...
   else
-    echo "Creating on Enterprise"
-    PAGES_URL="https://${ROOT_URL}/pages/${CLASS_ORG}/${STUDENT_REPO}/"
+    echo "Creating on GitHub Enterprise Server"
+    STUDENT_PAGES_URL="https://${ROOT_URL}/pages/${CLASS_ORG}/${STUDENT_REPO}/"
+    TEMPLATE_PAGES_URL="https://${ROOT_URL}/pages/${CLASS_ORG}/${STUDENT_REPO}/"
```
Comment (amyschoen) [inline script/create-game-repos:123]:
```suggestion
    TEMPLATE_PAGES_URL="https://${ROOT_URL}/pages/${CLASS_ORG}/${$TEMPLATE_REPO}/"
```

**Comment by parkerbxyz** [comment]
> You'll notice that both the student URL and the template URL are identical.

Whoops! Nice catch. Thanks, @amyschoen! ✨

**Comment by amyschoen** [comment]
Added suggestions on how to fix it.  The GHES version works for me, so I think the .com version should work for you.

**Comment by parkerbxyz** [comment]
I just tested this on github.com and it works as expected. 🚀

**Pull request merged**